### PR TITLE
#4979 [Manager] Event log receives a crazy amount of error messages w…

### PR DESCRIPTION
**Pull request by konekoya**
…hen there's no configurator available

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4979)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/hooks/connectorHooks.js [hunk 1]
```diff
@@ -142,18 +142,19 @@ export const useConnectorGroup = () => {
...
   const group = useConnectorGroup();
   const fetchConnectors = useFetchConnectorsAction();
   const isConnectorLoaded = useIsConnectorLoaded();
   const isConnectorLoading = useIsConnectorLoading();
+  const currentPipeline = hooks.usePipeline();
 
   useEffect(() => {
-    if (isConnectorLoaded || !isAppReady || isConnectorLoading) return;
+    if (isConnectorLoaded || isConnectorLoading) return;
+    if (!currentPipeline) return;
```
Comment (konekoya) [inline ohara-manager/client/src/hooks/connectorHooks.js:154]: @eechih @saivirtue This seems to fix the issue. I think it does make sense to only run this hook when _current pipeline_ is available. Do you guys think if there's a better way to fix this?

Comment (saivirtue) [reply ohara-manager/client/src/hooks/connectorHooks.js:154]: Hmm...This will delay the connector list to be loaded, right?

Comment (konekoya) [reply ohara-manager/client/src/hooks/connectorHooks.js:154]: Right. This seems to create another issue for not properly loading connector classes... I will update the PR

Comment (saivirtue) [reply ohara-manager/client/src/hooks/connectorHooks.js:154]: It looks like the event log keeps increasing the `/streams` and `/shabondis` APIs only. Maybe add the `!isAppReady` to those hooks will solve the problem?

Comment (konekoya) [reply ohara-manager/client/src/hooks/connectorHooks.js:154]: Didn't think about this... 😳  this does fix the issue. There are some unnecessary made by Shabondi though, i will fix it as well. Big thanks to you @saivirtue ❤️

Comment (konekoya) [reply ohara-manager/client/src/hooks/connectorHooks.js]: Updated :)

File: ohara-manager/client/src/store/reducers/ui/shabondi.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ const initialState = {
 export default function reducer(state = initialState, action) {
   switch (action.type) {
     case actions.fetchShabondis.REQUEST:
+    case actions.fetchConnectors.TRIGGER:
```
Comment (konekoya) [inline ohara-manager/client/src/store/reducers/ui/shabondi.js:28]: Prevent shabondi from making multiple requests

Comment (konekoya) [reply ohara-manager/client/src/store/reducers/ui/shabondi.js:28]: At the same time

Comment (saivirtue) [inline ohara-manager/client/src/store/reducers/ui/shabondi.js:28]: Could we add some comments here?

Comment (saivirtue) [reply ohara-manager/client/src/store/reducers/ui/shabondi.js:28]: Oops...I see there 😅

Comment (konekoya) [reply ohara-manager/client/src/store/reducers/ui/shabondi.js:28]: Yep. you mean something like: Ensure the loading flag is set to true when both request and trigger actions are fired?

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/src/store/reducers/ui/shabondi.js (hunk 2 above)
Comment (saivirtue) [reply ohara-manager/client/src/store/reducers/ui/shabondi.js:28]: Sounds great. Or the comment above is OK also:
> Prevent shabondi from making multiple requests at the same time

Comment (konekoya) [reply ohara-manager/client/src/store/reducers/ui/shabondi.js:31]: Added :)

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**